backend/carpool_request_service/carpool_request/app/carpool_request_application_service.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+
+from backend.user_service.user.domain.rider import Rider
+from backend.carpool_request_service.carpool_request.domain.carpool_request import CarpoolRequest
+from backend.common.rpc.infra.adapter.redis.redis_rpc_client import RedisRpcClient
+from backend.common.command.group_create_command import GroupCreateCommand, GROUP_CREATE_COMMAND
+
+
+class CarpoolRequestApplicationService():
+    
+    def create(self, from_location, to_location, minimum_passenger, rider_id):
+        rider = Rider.objects.get(id=rider_id)
+        result = CarpoolRequest.objects.create(from_location=from_location, to_location=to_location, \
+                                                minimum_passenger=minimum_passenger, rider=rider)
+    
+        hold_request = CarpoolRequest.objects.filter(status="IDLE")
+        same_location_request = hold_request.filter(from_location=result.from_location).filter(to_location=result.to_location)
+        if len(same_location_request) == 4:
```

Comment:
Does making a group currently work for 4 people only?

Reply:
Yes, now we consider 4 people case only.